bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -457,6 +469,65 @@ func (kfapp *coordinator) Apply(resources kftypes.ResourceEnum) error {
 	return nil
 }
 
+func (kfapp *coordinator) applyCommon() error {
+
+	// Get a K8s client
+	config := kftypes.GetConfig()
+	kubeclient := kftypes.GetClientset(config)
+
+	// Create KFApp's namespace
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: kfapp.KfDef.Namespace,
+		},
+	}
+	log.Infof("Creating namespace: %v", ns.Name)
```

Comment:
i understand why this has been moved into coordinator, but it still remains in gcp.
For the use case where --platform gcp is provided we would attempt install istio twice

Although this fixes the on-prem use-case, won't it break others?

Reply:
I didn't want to touch any GCP code in this PR, to make it as small as possible and thus easy to merge.
It also doesn't break anything.
`createNamespace` returns nil error if the Namespace is found and same with Istio.